src/cljs/main/broadfcui/utils.cljs

```diff
@@ -242,6 +242,9 @@
 (defn index-of [coll item]
   (first (keep-indexed (fn [i x] (when (= item x) i)) coll)))
 
+(defn first-matching [pred coll]
+  (first (keep #(when (pred %) %) coll)))
```

Comment:
This can just be `(first (filter pred coll))`

Reply:
can this even be `(some pred coll)`?

Reply:
No, that returns true/false

Reply:
Yup, that was the first thing I tried before Matt's suggestion, similarly disappointed

Reply:
```
cljs.user=> (some #(if (even? %) %) [1 2 3 4])
2
```
is working for me. I think it depends on the predicate? http://clojuredocs.org/clojure.core/some

Reply:
I think that would make it
```
(defn first-matching [pred coll]
  (some #(if (pred %) %) coll))
```
which seems less clear IMO